front/SettingsForm.tsx

```diff
@@ -106,6 +110,7 @@ const SettingsForm = () => {
           </fieldset>
           <button
             className="btn btn-lg btn-primary pull-xs-right"
+            data-cy="updateBtn"
```

Comment:
The `data-cy` attribute for the `updateBtn` button is correctly added, which is important for Cypress testing.